test/feat663-search.sh

```diff
@@ -10,6 +10,6 @@ fi
 if ! OUTPUT=$(${DUB} search dub -v 2>&1); then
     die $LINENO '`dub search dub` failed' "$OUTPUT"
 fi
-if ! grep -q '^dub (.*)\s'<<<"$OUTPUT"; then
-    die $LINENO '`grep -q '"'"'^dub (.*)\s'"'"'` failed' "$OUTPUT"
+if ! grep -q '^\s*dub (.*)\s'<<<"$OUTPUT"; then
+    die $LINENO '`grep -q '"'"'^\s*dub (.*)\s'"'"'` failed' "$OUTPUT"
```

Comment:
These adaptions shouldn't be necessary as it should be possible to detect that no terminal is available here. No?

Reply:
Sure, the output isn't colored, but I added two trailing spaces (like when you do `dub list`) to every line in the output of `dub search`. There's no reason why `dub lists` prints something like 

```
Packages known to dub:
  package-a 0.1.2
  package-b 0.2.4
```

while the results of `dub search` don't appear formatted the same way, IMHO. The regex change is to accomodate for this. Hope you understand.